Lib/test/test_typing.py

```diff
@@ -362,9 +362,9 @@ def test_etree(self):
         # See https://github.com/python/typing/issues/229
         # (Only relevant for Python 2.)
         try:
-            from xml.etree.cElementTree import Element
+            from xml.etree.ElementTree import Element
         except ImportError:
-            raise SkipTest("cElementTree not found")
+            raise SkipTest("ElementTree not found")
```

Comment:
I do not think it is possible.

Reply:
So how can I update? I would be very happy if you could help.

Reply:
Remove try/except.

Reply:
Thanks!